docs/_posts/examples/3400-01-26-game-controls.html

```diff
@@ -0,0 +1,81 @@
+---
+layout: example
+category: example
+title: Navigate the map with game-like controls
+description: Move around the map with game-like controls.
+---
+
+<style>
+    /* Dark attribution */
+    .mapboxgl-ctrl.mapboxgl-ctrl-attrib { background-color:rgba(0,0,0,0.5); }
+    .mapboxgl-ctrl.mapboxgl-ctrl-attrib a { color:#fff; }  
+
+    #instructions {
+        background-color: rgba(255, 255, 255, 0.1);
+        color: #fff;
+        padding: 10px;
+        border-radius: 3px;
+        position: absolute;
+        top: 50px;
+        width: 80%;
+        margin: 0 10%;
+        font-family: 'Open sans', sans-serif;
+        text-align: center;
+    }
+
+</style>
+
+<div id='map'></div>
+<div id='instructions'>Click on the map, then use the arrow keys to navigate the map.</div>
+
+
+<script>
+    //Initialize the map
+    var map = new mapboxgl.Map({
+        container: 'map',
+        style: 'mapbox://styles/mapbox/dark-v9',
+        center: [-87.6298, 41.8781],
+        zoom: 17,
+        bearing: -12,
+        pitch: 60,
+        interactive: false
+    });
+
+    var deltaDistance = 100; // how far the map moves in pixels when the up or down arrow is clicked
+    var deltaDegrees = 25; // rotation of the map in degrees when the left or right arrow is clicked
+
+    var instructions = document.getElementById('instructions');
+    map.on('click', function() {
+        instructions.style.display = 'none';
+    });
+
+    function easeFunction(t) {
+        return t * (2 - t);
+    }
+
+    document.body.addEventListener('keydown', function(e) {
+        e.preventDefault();
+        if (e.which === 38) { // up
+            map.panBy([0, -deltaDistance], {
+                easing: easeFunction
+            });
+        } else if (e.which === 40) { // down
+            map.panBy([0, deltaDistance], {
+                easing: easeFunction
+            });
+        } else if (e.which === 37) { // left
+            map.easeTo({
+                bearing: map.getBearing() - deltaDegrees,
+                easing: easeFunction,
+                animate: true
```

Comment:
also you can remove `animate: true` on your `map.eastTo()` calls -- its `true` by defailt 😸